spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -494,18 +494,26 @@ Then do this:
 
 This would upload the docker image to the docker hub in your custom repository.
 
-=== General FAQ on Spring Cloud Stream App Starters
+=== General faq on Spring Cloud Stream App Starters
 
 In the following sections, you can find a brief faq on various things that we discussed above and a few other infrastructure related topics.
 
 
 . What are Spring Cloud Stream Application Starters? +
-Spring Cloud Stream App Starters are individual repositories hosted under the GitHub organization - spring-cloud-stream-app-starters - on github (http://github.com/spring-cloud-stream-app-starters) +
+Spring Cloud Stream Application Starters are Spring Boot based Spring Integration applications that provide integration with external systems. GitHub: http://github.com/spring-cloud-stream-app-starters +
 Project page: http://cloud.spring.io/spring-cloud-stream-app-starters/
 
 
 . What is the parent for stream app starters? +
 The parent for all app starters is `app-starters-build` which is coming from the core project. https://github.com/spring-cloud-stream-app-starters/core
+For example:
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>app-starters-build</artifactId>
+        <version>1.2.0.BUILD-SNAPSHOT</version>
```

Comment:
I think it maybe possible to parameterize this as an asciidoctor variable so the version comes from properties in the maven plugin.